packages/fxa-content-server/app/scripts/views/sign_up_password.js

```diff
@@ -104,6 +123,27 @@ const SignUpPasswordView = FormView.extend({
   _doPasswordsMatch() {
     return this._getPassword() === this._getVPassword();
   },
+
+  _onConfirmPasswordKeyUp() {
+    if (!passwordCheckLock) {
+      passwordCheckLock = true;
```

Comment:
This changes the previous debounce behavior.  IIRC previously `debounce` is called on every keyup.  With this extra state, `debounce` is called on the first keyup, then again on the first keyup after `DELAY_BEFORE_PASSWORD_CHECK_MS`.  If the blur handler executes during the delay, it won't perform the check since `passwordCheckLock` will be true.

Is this intentional?

Reply:
Yea this was intentional because otherwise you would see a slight flicker on the tooltip when called twice.